addons/markers/XEH_postInit.sqf

```diff
@@ -1,20 +1,77 @@
-// by commy2
 #include "script_component.hpp"
 
-// recieve remote marker data
-[QGVAR(setMarkerNetwork), {_this call DFUNC(setMarkerNetwork)}] call EFUNC(common,addEventHandler);
+[QGVAR(loadMapData), DFUNC(loadMapData)] call EFUNC(common,addEventHandler);
+[QGVAR(setMarkerGlobal), DFUNC(setMarkerGlobal)] call EFUNC(common,addEventHandler);
+[QGVAR(loadBriefingMarkers), DFUNC(loadBriefingMarkers)] call EFUNC(common,addEventHandler);
+[QGVAR(onMapItemChange), DFUNC(onMapChange)] call EFUNC(common,addEventHandler);
 
-// recieve marker data for JIP
-[QGVAR(setMarkerJIP), {_this call DFUNC(setMarkerJIP)}] call EFUNC(common,addEventHandler);
+if(hasInterface) then {
```

Comment:
Most of the content of postInit should be wrapped in a `"settingsInitialized"` event handler, that quit inmediately if the feature is disabled:

```
["settingsInitialized", {
    if (!GVAR(enableUniqueMaps)) exitWith {};

    // All unit markers init and events here
    // ...
}] call EFUNC(common,addEventHandler);
```

That way you wouldn't have to constantly check for `!GVAR(enableUniqueMaps)` anymore, and there would be no code running if the feature is disabled. 

Check other modules for examples on this pattern (e.g. ace_hearing)